examples/bevy_ggrs/src/box_game.rs

```diff
@@ -124,35 +123,41 @@ pub fn setup_scene_system(
         transform.translation.z = z;
         let color = PLAYER_COLORS[handle % PLAYER_COLORS.len()];
 
+        // Entities which will be rolled back can be created just like any other...
         commands
             .spawn((
+                // ...add visual information...
                 PbrBundle {
                     mesh: meshes.add(Mesh::from(shape::Cube { size: CUBE_SIZE })),
                     material: materials.add(color.into()),
                     transform,
                     ..default()
                 },
+                // ...flags...
                 Player { handle },
+                // ...and components which will be rolled-back...
                 Velocity::default(),
             ))
+            // ...just ensure you call `add_rollback()`
+            // This ensures a stable ID is available for the rollback system to refer to
             .add_rollback();
     }
 
     // light
     commands.spawn(PointLightBundle {
-        transform: Transform::from_xyz(-4.0, 8.0, 4.0),
+        transform: Transform::from_xyz(4.0, 8.0, 4.0),
         ..default()
     });
     // camera
-    for mut transform in camera_query.iter_mut() {
-        *transform = Transform::from_xyz(-2.0, 2.5, 5.0).looking_at(Vec3::ZERO, Vec3::Y);
-    }
+    commands.spawn(Camera3dBundle {
+        transform: Transform::from_xyz(0.0, 7.5, 0.5).looking_at(Vec3::ZERO, Vec3::Y),
+        ..default()
+    });
```

Comment:
I don't like the perspective/lighting in the bevy_ggrs example, so I changed it for matchbox... My brain gets kind of confused looking at almost top-down perspective. It's kind of a subjective thing, which is why I haven't made a PR for it in bevy_ggrs, but I think it made for better demo posts on social media.

Reply:
Fair enough, I'll revert that change!